src/components/sbb-form-field/sbb-form-field.scss

```diff
@@ -5,8 +5,9 @@
 
 :host {
   display: inline-block;
-  width: #{px-to-rem-build(300)};
+  max-width: #{px-to-rem-build(300)};
```

Comment:
The width is still 300px, but this make the form field shrink in smaller breakpoint situations

Reply:
Could we maybe use `width: max(#{px-to-rem-build(300)}, 100%);` instead of max-width? I would prefer if consumers only need to override one property to change the size.

Reply:
unfortunately no, because max() sets the largest possible between the two values, going over 300px
but I understand the point, I see if there is a solution that involves only one property

Reply:
Maybe using `min` instead of `max`. I always confuse them on how to use them in CSS.

Reply:
exactly :)